launch_testing/test/launch_testing/test_resolve_process.py

```diff
@@ -66,3 +71,120 @@ def test_unlaunched_process_lookup_with_substitutions(self):
         # Since the process wasn't launched yet, and it has substitutions that need to be
         # resolved by the launch system, we won't be able to take a guess at the command
         self.assertIn('Unknown', str(cm.exception))
+
+
+class TestStringProcessResolution(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # Run a launch so we can get some real looking proc_info and proc_output objects to test
+        # against
+        proc_command = os.path.join(
+            ament_index_python.get_package_prefix('launch_testing'),
```

Comment:
@pbaughman Windows is barfing at this one, as we're trying to run a Python script that is not properly wrapped to be treated as an executable on said platform (as it would be if we were installing it as a `console_script`). A workaround is to use `python` as your actual command, as in https://github.com/ros2/launch/blob/master/launch_testing/test/launch_testing/test_io_handler_and_assertions.py#L35.

Reply:
I understand.  I'll be back on Tuesday to fix this.  Maybe I'll serialize a known proc_info object instead of generating one on the fly